docs/application_configuration.rst

```diff
@@ -0,0 +1,140 @@
+Application configuration
+=========================
+This section will describe what an application is and how to create and configure it. In QNE, an application functions
+as a sort of 'template' to create your experiment for. You decide how many roles you want, what type of inputs and the
+actual functionality of your application.
+
+The creation and configuration of your application is split into three parts. First, you will create your application,
+then you can follow the config and src pages to configure the contents of your application and make it ready for an
+experiment.
+
+
+Create application
+------------------
+Execute the following command to create an application in your current directory:
+
+.. code-block:: console
+
+    qne application create [OPTIONS] APPLICATION_NAME ROLES...
+
+    Arguments:
+      APPLICATION_NAME The name of the application [required]
+      ROLES            Names of the roles to be created [required]
+
+
+This could be an example of what this command could look like:
+
+.. code-block:: console
+
+    qne application create teleport Sender Receiver
+
+
+You can add more than two roles as far as the network allows. If your network consists of five nodes, it is not
+possible to create more than five roles. In `Network Information <network_information.html>`_ you can find the number
+of nodes a network has. Your application will only work on networks that contains enough nodes for the number of roles
+defined for the application.
+
+After executing this command, you should have a directory structure that looks like this:
+
+.. image:: images/application_structure.png
```

Comment:
How did you create this picture? A tool or by hand?

Reply:
I first wanted to create a tree structure either manually or with a library, but couldn't find anything of rts. The picture is made using a snipping tool.